tests/integration/components/select-list-test.js

```diff
@@ -183,3 +183,15 @@ test('binds properly to `title` attribute', function(assert) {
 
   assert.equal(this.$('select').attr('title'), title);
 });
+
+test('remains invalid when prompt is selected', function(assert) {
+  var options = [];
+  this.set('options', options);
+  this.set('prompt', 'First prompt');
+
+  this.render(hbs`
+    {{select-list content=options value=selection prompt=prompt}}
+  `);
+
+  assert.equal(this.$('select option:selected')[0].validity.valid, false);
```

Comment:
This fails in chrome (or phantomjs) and only seems to be working in Firefox. If you have another idea on how to check for valid state, go with it.

Alternatively, if there is no other way, we can just check to make sure the selected value of the select element itself is set to nothing.

```
assert.equal(this.$('select').val(), null);
```

That should be good enough.